doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -110,41 +113,53 @@ The presence of a ``bicr`` node in the application devicetree will automatically
 Flashing the BICR binary
 ========================
 
+##TODO
+
 After the |NCS| build system generates the BICR binary, you must flash this binary manually.
 The content of BICR should be loaded to the SoC only once and should not be erased nor modified unless the PCB layout changes.
-To manually flash the generated :file:`bicr.hex` file to the SoC, use nRF Util as follows::
+To manually program the generated :file:`bicr.hex` file to the SoC, use nRF Util as follows::
+
+    nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware bicr.hex --core Application --serial-number <serial_number>
+
+You only need to follow this programming process once, assuming the PCB configuration applied through the BICR is correct the first time.
+However, it is also possible to reprogram the BICR while in the LCS ``RoT``.
+This can be useful, for example, when adjusting the configuration as the PCB design gets refined or modified, requiring the process to be repeated.
+
+Validate the BICR binary
+------------------------
+
+After programming the BICR binary onto the device, you should validate whether the BICR was successfully installed and executed.
 
-    nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware bicr.hex` --core Secure --serial-number <serial_number>
+Reset the device
```

Comment:
If I program the BICR and then go to this step, then I want to validate it.
Then I come to this step, and I see "ah okay I reset the device".
Now if my BICR is wrong, and the device is not operational, then I run the chance that I cannot recover it right?
Would it not be better to describe the nrfutil method before the reset method, because the nrfutil method below assumedly does run the risc or breaking the device?

Reply:
Since the device needs to be reset to do the seconds step, I close this comment.
See https://github.com/nrfconnect/sdk-nrf/pull/16796#discussion_r1745497275

Reply:
Maybe I can explain a bit better.

We have no way to test the BICR, except for programming and loading it through a reset, then seeing what happens.

So the procedure is:
1. We program the BICR
2. We reset the chip so that the System Controller ROM code can apply the power settings
3. We double check that the System Controller ROM successfully applied the power settings

So the actual customer procedure I would suggest is:
1. Program the BICR
2. Reset the chip
3. `nrfutil device x-read --address 0x2F88FF1C --serial-number 1051164514 --core Secure`
We then look at what happens during step 3:
 - Error: Device is likely trash :(
 - `0xCE68C97C` (BICR loading failed): You _should_ be able to program another BICR
 - `0xD78213DF` (No BICR present): Customer did not actually program a BICR, try again :)
 - `0x289CFB73` (Success): BICR settings were applied and system appears to behave nominally. Proceed with bringup

Note:
This will _only_ validate the parts of the BICR relating to power as the clock section is not applied by ROM code, but rather by System Controller FW, starting in ROT. However, the clock part being incorrect _should_ still allow you to replace the BICR.

The reason we should at least in part validate the BICR here, is that if we do not, then the failure may not become apparent until after the LCS transition to ROT, in which case it gets very hard to prove that the issue is with the BICR, and not with SDSC, or some other part of the system. We can also only read this ROM report in Empty, so its a good idea to do it here.

Reply:
I guess sysrom might also apply tamper controller and shield settings, @David-Garcia-Polo would have to answer that, as I have never played around with those things.